.github/workflows/test.yml

```diff
@@ -275,6 +275,21 @@ jobs:
             --jsonfile "$REPORTS_PATH/$NAME.json" \
             --packages="$PACKAGES" \
             -- ${{ matrix.go_test_flags || '' }}
+      - name: Modify junit.xml for BuildPulse
+        if: always()
+        run: |
+          # Modify test suite name and classname attributes in JUnit XML for better grouping
+          # in BuildPulse. itests are run with go test ./itests/file_test.go and therefore Go
+          # assigns the name and classname attributes to "command-line-arguments". Others get the
+          # package name for both.
+          if [[ "${{ matrix.name }}" == itest-* ]]; then
+            PACKAGE_NAME=$(basename "$PACKAGES" .go)
+            sed -i 's/ name="command-line-arguments"/ name="itests"/g' "$REPORTS_PATH/$NAME.xml"
+            sed -i 's/classname="command-line-arguments"/classname="'"$PACKAGE_NAME"'"/g' "$REPORTS_PATH/$NAME.xml"
+          else
+            sed -i 's# name="github.com/filecoin-project/lotus/\(.*\)"# name="'${{ matrix.name }}':\1"#g' "$REPORTS_PATH/$NAME.xml"
+          fi
+          cat "$REPORTS_PATH/$NAME.xml"
```

Comment:
check out the actions run output if you want to see what we're dealing with here

Reply:
Ack - this makes sense to me.  I'll resolve this so we can merge.